tests/rptest/tests/node_operations_fuzzy_test.py

```diff
@@ -272,6 +272,7 @@ def add_node(idx, cleanup=True):
                                          preserve_logs=True)
             # we do not reuse previous node ids and override seed server list
             self.redpanda.start_node(self.redpanda.get_node(idx),
+                                     timeout=60,
```

Comment:
Maybe this should be defined as an expression that adds of the minimum inter-failure delay plus the max failure duration in failure_injector_loop?

(to avoid this becoming flaky again if the failure params are tweaked to disable the cluster for longer periods)